src/Todo/DB.hs

```diff
@@ -0,0 +1,49 @@
+module Todo.DB
+    ( TodoDB (..)
+    , DBError (..)
+    , addTodo
+    , addTag
+    , deleteTag
+    , editTag
+    ) where
+
+import           Todo.Model
```

Comment:
Well, I don't insist on formatting your code, but it's better to put imports from your project under different section after empty line after imports from packages. Just to make it more visible what is exported from Hackage and what is yours. Also, please, use explicit import list here... Please... Please..